topology/nhlt/nhlt-processor.c

```diff
@@ -316,6 +329,11 @@ int nhlt_set_dai(struct tplg_pre_processor *tplg_pp,
 	tplg_pp_debug("nhlt_set_dai id %s", id);
 
 	/* set dai parameters here */
+	if (!strncmp(id, "DMIC", 4)) {
+		ret = nhlt_dmic_set_params(tplg_pp, cfg, parent);
```

Comment:
what does cfg point to here?

Reply:
does nhlt_dmic_set_params() end up with generating the blob? Its hard to tell where the blob generation is happening

Reply:
cfg should be the DAI config from topology?

Reply:
I'm getting the dai params from topology2 parsing (with auto_attr) -> setting the values to internal structs -> all blobs are generated at the end of parsing from these internal structs. Sof fw code is doing it like this -> I don't want to start to modify too much for e.g. the dmic blob code -> will differ from sof too much -> horrible amount of possible errors for me